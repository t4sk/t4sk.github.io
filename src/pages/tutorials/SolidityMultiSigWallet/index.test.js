import React from "react"
import { shallow } from "enzyme"
import { SolidityMultiSigWallet } from "./index"

const props = {
  match: {
    url: "localhost:3000/tutorials/solidity-multi-sig-wallet",
  },
}

beforeEach(() => {})

test("it renders", () => {
  const component = shallow(<SolidityMultiSigWallet {...props} />)

  expect(component).toMatchSnapshot()
})
