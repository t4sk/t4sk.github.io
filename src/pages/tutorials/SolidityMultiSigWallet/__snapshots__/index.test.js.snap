// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<PageLayout>
  <SEO
    description="Learn how to program smart contracts."
    keywords={
      Array [
        "smart contract",
        "solidity",
        "ethereum",
      ]
    }
    lang="en"
    meta={Array []}
    title="Build a Multi-Sig Wallet in Solidity | Smart Contract Programmer"
  />
  <Container>
    <div
      className="component"
    >
      <h1
        className="header"
      >
        Build a Multi-Sig Wallet in Solidity
      </h1>
      <div
        className="section"
      >
        <Logo />
        <div
          className="lessons"
        >
          <h2>
            Solidity Contract
          </h2>
          <List
            divided={true}
            relaxed={true}
            selection={true}
          >
            <ListItem
              key="0"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/00-intro"
                >
                  Intro
                </ListHeader>
              </ListContent>
            </ListItem>
            <ListItem
              key="1"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/01-constructor-exercise"
                >
                  Constructor
                </ListHeader>
                <ListDescription>
                  Exercise
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="2"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/01-constructor-solution"
                >
                  Constructor
                </ListHeader>
                <ListDescription>
                  Solution
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="3"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/02-fallback-exercise"
                >
                  Fallback
                </ListHeader>
                <ListDescription>
                  Exercise
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="4"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/02-fallback-solution"
                >
                  Fallback
                </ListHeader>
                <ListDescription>
                  Solution
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="5"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/03-submit-transaction-exercise"
                >
                  Submit Transaction
                </ListHeader>
                <ListDescription>
                  Exercise
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="6"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/03-submit-transaction-solution"
                >
                  Submit Transaction
                </ListHeader>
                <ListDescription>
                  Solution
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="7"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/04-confirm-transaction-exercise"
                >
                  Confirm Transaction
                </ListHeader>
                <ListDescription>
                  Exercise
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="8"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/04-confirm-transaction-solution"
                >
                  Confirm Transaction
                </ListHeader>
                <ListDescription>
                  Solution
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="9"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/05-execute-transaction-exercise"
                >
                  Execute Transaction
                </ListHeader>
                <ListDescription>
                  Exercise
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="10"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/05-execute-transaction-solution"
                >
                  Execute Transaction
                </ListHeader>
                <ListDescription>
                  Solution
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="11"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/06-revoke-confirmation-exercise"
                >
                  Revoke Confirmation
                </ListHeader>
                <ListDescription>
                  Exercise
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="12"
            >
              <ListIcon
                color="black"
                name="code"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/06-revoke-confirmation-solution"
                >
                  Revoke Transaction
                </ListHeader>
                <ListDescription>
                  Solution
                </ListDescription>
              </ListContent>
            </ListItem>
            <ListItem
              key="13"
            >
              <ListIcon
                color="red"
                name="youtube"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/07-demo"
                >
                  Demo
                </ListHeader>
              </ListContent>
            </ListItem>
            <ListItem
              key="14"
            >
              <ListIcon
                color="black"
                name="github"
                size="large"
                verticalAlign="middle"
              />
              <ListContent>
                <ListHeader
                  as="a"
                  href="localhost:3000/tutorials/solidity-multi-sig-wallet/08-code"
                >
                  Code
                </ListHeader>
              </ListContent>
            </ListItem>
          </List>
        </div>
      </div>
    </div>
  </Container>
</PageLayout>
`;
